shaq_overflow/app/controllers/answers_controller.rb

```diff
@@ -0,0 +1,54 @@
+class AnswersController < ApplicationController
+
+  def index
+    @answers = Answer.all
+  end
+
+  def show
+    @answer = Answer.find(params[:id])
+  end
+
+  def new
+    @answer = Answer.new
+    @question = Question.find(params[:question_id])
+  end
+
+  def create
+    @answer = Answer.new(answer_params)
+      #***************************
+      #CHANGE THIS TO CURRENT USER
+      #***************************
+    @answer.user = User.find(1)
+    @answer.question = Question.find(params[:question_id])
+    if @answer.save
```

Comment:
You could chain the build here and do
@question = Question.find(params[:question_id])
@question.answers.build(answer_params)
if @question.save ...